src/screens/profile/components/user-info/user-info.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { View } from 'react-native';
+import { Avatar } from 'react-native-paper';
+
+import { Heading, Text } from 'src/components';
+import { AppColor, HeadingLevel } from 'src/common/enums';
+
+import styles from './styles';
+
+const UserInfo = () => {
+  return (
+    <View style={styles.container}>
+      <Avatar.Image
+        size={148}
+        source={{
+          uri: 'https://cdn.iconscout.com/icon/free/png-256/avatar-370-456322.png',
+        }}
+        style={styles.avatar}
+      />
+      <View style={styles.info}>
+        <Heading level={HeadingLevel.H4} style={styles.name}>
+          Cristofer Westervelt
+        </Heading>
+        <Heading level={HeadingLevel.H6} style={styles.role}>
```

Comment:
Here we have two options: use Heading and change font-family to 'NunitoSans-Regular' or use just text and set fontSize to 16. I would prefer the second one.
```suggestion
        <Text style={styles.role}>
```